cider/tests/nextgen/compiler/NextgenCompilerTest.cpp

```diff
@@ -111,9 +113,36 @@ TEST_F(NextgenCompilerTest, FrameworkTest) {
   executeTest("select a + b, a - b from test where a < b");
 }
 
+class CiderNextgenCompilerTestBase : public CiderTestBase {
+ public:
+  CiderNextgenCompilerTestBase() {
+    table_name_ = "test";
+    create_ddl_ = "CREATE TABLE test(col_1 BIGINT NOT NULL, col_2 BIGINT NOT NULL)";
+    QueryArrowDataGenerator::generateBatchByTypes(
+        schema_,
+        array_,
+        99,
+        {"col_1", "col_2"},
+        {CREATE_SUBSTRAIT_TYPE(I64), CREATE_SUBSTRAIT_TYPE(I64)});
+  }
+};
+
+TEST_F(CiderNextgenCompilerTestBase, integerFilterTest) {
+  assertQueryArrow("SELECT col_1 + col_2 FROM test WHERE col_1 < col_2");
```

Comment:
Seems this TC not actually trigger nextgen?

Reply:
will trigger when run with ctest, otherwise require run with --use-nextgen-compile